docs/install_guide.adoc

```diff
@@ -0,0 +1,206 @@
+| 139b5c649f074eae874ca2d8a3b7fdcb | regionOne | glance       | image        | True    | public    | https://glance-default-public-openstack.apps-crc.testing |
+| 1c6ff2b6f2b54a13a8fefc9dacb8f4a9 | regionOne | neutron      | network      | True    | public    | https://neutron-public-openstack.apps-crc.testing        |
+| 4bb6dde288a64c08bc2cd8e2d714aa6d | regionOne | glance       | image        | True    | internal  | http://glance-default-internal.openstack.svc:9292        |
+| 6dfd4ac8fae2485580f626d743603c05 | regionOne | keystone     | identity     | True    | public    | https://keystone-public-openstack.apps-crc.testing       |
+| 98e631462d2f4828a55e558f36e23e2f | regionOne | neutron      | network      | True    | internal  | http://neutron-internal.openstack.svc:9696               |
+| 9a6be9fccf3e42cd8b3205a9c1d88114 | regionOne | placement    | placement    | True    | internal  | http://placement-internal.openstack.svc:8778             |
+| aa4363e8f74641ce9a53c14fd4ce8184 | regionOne | placement    | placement    | True    | public    | https://placement-public-openstack.apps-crc.testing      |
+| b313c0ce41b840a884b8e0ac4a5a819f | regionOne | cinderv3     | volumev3     | True    | public    | https://cinder-public-openstack.apps-crc.testing/v3      |
+| ccadeae4322346fe9feee6bb0677b014 | regionOne | nova         | compute      | True    | internal  | http://nova-internal.openstack.svc:8774/v2.1             |
+| dfac9cb0706d411b8c90c6df264fb575 | regionOne | nova         | compute      | True    | public    | https://nova-public-openstack.apps-crc.testing/v2.1      |
+| e8314601b5184f74b99b0815e1d57d5e | regionOne | cinderv3     | volumev3     | True    | internal  | http://cinder-internal.openstack.svc:8776/v3             |
++----------------------------------+-----------+--------------+--------------+---------+-----------+----------------------------------------------------------+
+----
+
+Now verify that Telemetry operator with Prometheus metric storage is ready:
+
+[,console]
+----
+$ oc get telemetry
+NAME        STATUS   MESSAGE
+telemetry   True     Setup complete
+
+$ oc get metricstorage
+NAME             STATUS   MESSAGE
+metric-storage   True     Setup complete
+
+$ oc get route metric-storage-prometheus
+NAME                        HOST/PORT                                              PATH   SERVICES                    PORT   TERMINATION     WILDCARD
+metric-storage-prometheus   metric-storage-prometheus-openstack.apps-crc.testing          metric-storage-prometheus   web    edge/Redirect   None
+----
+
+Prometheus metrics will be viewable in a web browser at the `HOST/PORT` address returned above. ie:
+https://metric-storage-prometheus-openstack.apps-crc.testing.
+
+== Installing the Operator
+
+.Procedure
+
+. Now that you have a ready working environment, you can install the Watcher Operator.
```

Comment:
I would suggest moving the first sentence immediately after the heading:

== Installing the Operator
Now that you have a ready working environment, you can install the Watcher Operator.